Misc/python.man

```diff
@@ -183,9 +182,11 @@ useful to inspect global variables or a stack trace when a script
 raises an exception.
 .TP
 .B \-I
-Run Python in isolated mode. This also implies \fB\-E\fP, \fB\-P\fP and \fB\-s\fP. In
-isolated mode sys.path contains neither the script's directory nor the user's
-site-packages directory. All PYTHON* environment variables are ignored, too.
+Run Python in isolated mode.
+This also implies \fB\-E\fP, \fB\-P\fP and \fB\-s\fP.
+In isolated mode sys.path contains neither the script's directory nor the
+user's site\-packages directory.
+All PYTHON* environment variables are ignored, too.
```

Comment:
This looks mostly as reformatting.  Please revert everything, except real change (dash escape in site-packages).